key4hep-build/action.yml

```diff
@@ -99,7 +99,7 @@ runs:
           cd /${name}
           # There is no k4_local_repo in the LCG stacks
           k4_local_repo || true
-          mkdir build
+          mkdir -p build
```

Comment:
Was this needed? It should always work except if there is a build folder already, in which case well I guess it depends on the repo but building may not work (not a problem for Key4hep repos).

Reply:
It's necessary because if there is external data that is cached, it will create the `<prefix>/build/ExternalData` directory.